shuup_tests/front/test_general_template_helpers.py

```diff
@@ -52,19 +53,34 @@ def test_get_listed_products_orderable_only():
 
     from shuup.front.template_helpers import general
 
-    assert len(general.get_listed_products(context, n_products, orderable_only=True)) == 0
-    assert len(general.get_listed_products(context, n_products, orderable_only=False)) == 1
+    cache.clear()
```

Comment:
Summarize a little what is going here. We shouldn't need to clear caches inside one test unless we are just testing something specific with the cache.

Reply:
Ok, I've cleared some too much `cache.clear()`. The point is. As caches are not bumped into a same test, the cache will return invalid objects. If you see a test case, it checks that the method doesn't return nothing, that an order is created and the same function is called again. The cache will take place and return wrong values. I am not trying to test the cache, I am testing that the function will return the expected values. We create orders and add categories directly in code, so caches are not bumped.

We have also this issue that I created when I was changing this file: https://github.com/shuup/shuup/issues/1347
We NEVER bump these caches, this is a bug. We should probably introduce some provide or signal to bump them.

You can also see that I run a `for cache_test in range(2):` in test, so the first loop is to calculate the actual value, and the second to test the cache is properly working.